harvest/algo.py

```diff
@@ -37,7 +37,7 @@ def remove_symbol(self, symbol: str) -> None:
     
     ############ Functions interfacing with broker #################
 
-    def buy(self, *args, **kwargs):
+    def buy(self, symbol: str=None, quantity: int=0, in_force: str='gtc', extended: bool=False):
```

Comment:
replace args kwargs with the full list of parameters

Reply:
Same should be done for sell, buy_await, sell_await, buy_option and sell_option

Reply:
it had, see the other changes of this file. I did not write comments for all of them.